diff --git a/src/wp/main/unix/GNUmakefile.am b/src/wp/main/unix/GNUmakefile.am
--- a/src/wp/main/unix/GNUmakefile.am
+++ b/src/wp/main/unix/GNUmakefile.am
@@ -51,7 +51,7 @@ ABIWORD_SODIR = $(shell pwd)
 ABIWORD_DYNBINARY = abiword-dynamic
 
 $(ABIWORD_LIBRARY): $(ABI_LIBS) $(srcdir)/libabiword.cpp $(srcdir)/../xp/abi_ver.cpp
-	$(CXX) -shared -Wl,-soname,$(ABIWORD_LIBRARY) -o $@ 		\
+	$(CXX) -shared -Wl,-soname,$(ABIWORD_LIBRARY).0 -o $@ 		\
 		$(CPPFLAGS) $(CXXFLAGS) $(INCLUDES)			\
 		-DPACKAGE="\"$(PACKAGE)\""				\
 		$(srcdir)/../xp/abi_ver.cpp				\
@@ -69,13 +69,14 @@ $(ABIWORD_DYNBINARY): $(ABIWORD_LIBRARY) $(srcdir)/UnixMain.cpp
 		$(PLATFORM_LIBS) $(OTHER_LIBS) 		\
 		-L$(ABIWORD_SODIR) -labiword-$(ABIWORD_SERIES)
 
-CLEANFILES += $(ABIWORD_DYNBINARY) $(ABIWORD_LIBRARY)
+CLEANFILES += $(ABIWORD_DYNBINARY) $(ABIWORD_LIBRARY).0 $(ABIWORD_LIBRARY)
 ALL_LOCAL_DEPS += $(ABIWORD_DYNBINARY)
 INSTALL_EXEC_LOCAL_DEPS += install-libabiword
 
 install-libabiword: 
 	@mkdir -p $(DESTDIR)/$(libdir)
-	cp $(ABIWORD_LIBRARY) $(DESTDIR)/$(libdir)/$(ABIWORD_LIBRARY)
+	install -T $(ABIWORD_LIBRARY) $(DESTDIR)/$(libdir)/$(ABIWORD_LIBRARY).0
+	ln -sf $(ABIWORD_LIBRARY).0 $(DESTDIR)/$(libdir)/$(ABIWORD_LIBRARY)
 
 endif
 
