diff --git a/src/wp/main/unix/GNUmakefile.in b/src/wp/main/unix/GNUmakefile.in
--- a/src/wp/main/unix/GNUmakefile.in
+++ b/src/wp/main/unix/GNUmakefile.in
@@ -44,7 +44,7 @@ DIST_COMMON = $(srcdir)/GNUmakefile.am $(srcdir)/GNUmakefile.in \
 @WITH_SYSTEM_GOFFICE_FALSE@am__append_4 = @GOFFICE_LIBS@
 @ENABLE_SPELL_TRUE@am__append_5 = $(top_builddir)/src/other/spell/xp/libSpell.a
 bin_PROGRAMS = abiword$(EXEEXT)
-@WITH_LIBABIWORD_TRUE@am__append_6 = $(ABIWORD_DYNBINARY) $(ABIWORD_LIBRARY)
+@WITH_LIBABIWORD_TRUE@am__append_6 = $(ABIWORD_DYNBINARY) $(ABIWORD_LIBRARY).0 $(ABIWORD_LIBRARY)
 @WITH_LIBABIWORD_TRUE@am__append_7 = $(ABIWORD_DYNBINARY)
 @WITH_LIBABIWORD_TRUE@am__append_8 = install-libabiword
 subdir = src/wp/main/unix
@@ -741,7 +741,7 @@ uninstall-am: uninstall-binPROGRAMS uninstall-headersDATA
 #@PLATFORM@/t/*.o 
 
 @WITH_LIBABIWORD_TRUE@$(ABIWORD_LIBRARY): $(ABI_LIBS) $(srcdir)/libabiword.cpp $(srcdir)/../xp/abi_ver.cpp
-@WITH_LIBABIWORD_TRUE@	$(CXX) -shared -Wl,-soname,$(ABIWORD_LIBRARY) -o $@ 		\
+@WITH_LIBABIWORD_TRUE@	$(CXX) -shared -Wl,-soname,$(ABIWORD_LIBRARY).0 -o $@ 		\
 @WITH_LIBABIWORD_TRUE@		$(CPPFLAGS) $(CXXFLAGS) $(INCLUDES)			\
 @WITH_LIBABIWORD_TRUE@		-DPACKAGE="\"$(PACKAGE)\""				\
 @WITH_LIBABIWORD_TRUE@		$(srcdir)/../xp/abi_ver.cpp				\
@@ -761,7 +761,8 @@ uninstall-am: uninstall-binPROGRAMS uninstall-headersDATA
 
 @WITH_LIBABIWORD_TRUE@install-libabiword: 
 @WITH_LIBABIWORD_TRUE@	@mkdir -p $(DESTDIR)/$(libdir)
-@WITH_LIBABIWORD_TRUE@	cp $(ABIWORD_LIBRARY) $(DESTDIR)/$(libdir)/$(ABIWORD_LIBRARY)
+@WITH_LIBABIWORD_TRUE@	install -T $(ABIWORD_LIBRARY) $(DESTDIR)/$(libdir)/$(ABIWORD_LIBRARY).0
+@WITH_LIBABIWORD_TRUE@	ln -sf $(ABIWORD_LIBRARY).0 $(DESTDIR)/$(libdir)/$(ABIWORD_LIBRARY)
 
 ###
 
