Author: Dmitry Smirnov <onlyjob@member.fsf.org>
Last-Update: 2012-08-31
Forwarded: no
Description: correct libgoffice-0.10 version requirements

--- a/configure.in
+++ b/configure.in
@@ -93,7 +93,7 @@
 # optional deps
 enchant_req='enchant >= 1.2.0'
 gio_req='gio-2.0'
-goffice_req='libgoffice-0.10 >= 0.10.0'
+goffice_req='libgoffice-0.10 >= 0.9.1'
 
 hildon_pkgs='hildon-1 hildon-fm-2 dbus-1 libosso'
 dnl libosso-gsf-1'
--- a/plugins/goffice/plugin.m4
+++ b/plugins/goffice/plugin.m4
@@ -11,7 +11,7 @@
   goffice_req=libgoffice-0.10
 fi
 
-goffice_pkgs="$goffice_req >= 0.10.0"
+goffice_pkgs="$goffice_req >= 0.9.1"
 goffice_deps="no"
 
 if test "$enable_goffice" != ""; then
