Last-Update: 2012-08-24
Author: Dmitry Smirnov <onlyjob@member.fsf.org>
Forwarded: not-needed
Description: backported patch (r31768)
 Fix bug: correct horizontal scroll limit

--- a/src/wp/ap/gtk/ap_UnixFrame.cpp
+++ b/src/wp/ap/gtk/ap_UnixFrame.cpp
@@ -320,7 +320,7 @@
 	// (with clamping).  then cause the view to scroll.
 
 	gfloat xoffNew = xoff;
-	gfloat xoffMax = gtk_adjustment_get_upper(pFrameImpl->m_pVadj) - 
+	gfloat xoffMax = gtk_adjustment_get_upper(pFrameImpl->m_pHadj) - 
 		gtk_adjustment_get_page_size(pFrameImpl->m_pHadj);
 	if (xoffMax <= 0)
 		xoffNew = 0;
