Last-Update: 2012-09-24
Author: Dmitry Smirnov <onlyjob@member.fsf.org>
Forwarded: not-needed
Origin: r31855
Bug-Abiword: http://bugzilla.abisource.com/show_bug.cgi?id=13405
Description: backported patch to fix GTK3 bug

--- a/src/wp/ap/gtk/abiwidget.cpp
+++ b/src/wp/ap/gtk/abiwidget.cpp
@@ -2022,7 +2022,7 @@
 	gtk_widget_set_can_focus(GTK_WIDGET(abi), true);
 	gtk_widget_set_receives_default(GTK_WIDGET(abi), true);
 	gtk_widget_set_can_default(GTK_WIDGET(abi), true);
-	gtk_widget_set_has_window(GTK_WIDGET(abi), false);
+	gtk_widget_set_has_window(GTK_WIDGET(abi), true);
 }
 
 static void
